### PR TITLE
#110 input data types

### LINKED ISSUE #110: Fix input data types
Test data was using numbers as all values, where as the server feeds strings, causing issues with generating values within the requisition lines

### PR DESCRIPTION
**Pull request by joshxg**
fixes #110

### CODE REVIEW

File: src/utilities/requests/transformations.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import parseDate from "../ui/parseDate";
+
+export default function requisitionResponse(data) {
```
Comment (wlthomson) [inline src/utilities/requests/transformations.js]: Just to keep consistency, can this use arrow syntax?

File: src/ui/containers/RequisitionView.js [hunk 2]
```diff
@@ -132,24 +132,64 @@ class RequisitionView extends React.Component {
...
+    const closing = Math.max(previous + received - consumed + adjustments, 0);
+    const suggested = Math.max(daysToSupply * consumed - closing, 0);
+    updatedLine.stockSuggested = suggested;
+    updatedLine.stockClosing = closing;
+    requisitionLines[index] = updatedLine;
+  };
+
+  validateInput = (stringToTest, regexPattern) => {
+    const regex = {
+      MOS: /^(\d{1,2}|\d{2}\.\d{1,2})$/m,
```
Comment (wlthomson) [inline src/ui/containers/RequisitionView.js]: It's a bit verbose, but can we use `MONTHS_OF_SUPPLY,` or `MONTHS_SUPPLY` here? @andreievg advised me to  avoid abbreviations wherever possible.

File: src/utilities/requests/transformations.js (hunk 1 above)
Comment (joshxg) [reply src/utilities/requests/transformations.js]: done

File: src/ui/containers/RequisitionView.js (hunk 2 above)
Comment (joshxg) [reply src/ui/containers/RequisitionView.js]: cool, done

**Pull request merged**